projects/demo-playwright/tests/kit/input-month/input-month.spec.ts

```diff
@@ -0,0 +1,39 @@
+import {TuiDocumentationPagePO, tuiGoto, TuiInputMonthPO} from '@demo-playwright/utils';
+import {expect, Locator, test} from '@playwright/test';
+
+const {describe, beforeEach} = test;
+
+describe(`InputMonth`, () => {
+    describe(`API`, () => {
+        let documentationPage: TuiDocumentationPagePO;
+        let example: Locator;
+        let inputMonth: TuiInputMonthPO;
+
+        test.use({
+            viewport: {width: 360, height: 500},
+        });
+
+        beforeEach(async ({page}) => {
+            await tuiGoto(page, `components/input-month/`);
```

Comment:
Probably we can drop this line

(Locators can initialized even before navigation)

Reply:
```suggestion
```